client/layout/masterbar/style.scss

```diff
@@ -456,6 +457,10 @@ $autobar-height: 20px;
 	}
 }
 
+.masterbar-cart-button .gridicon + .masterbar__item-content {
+	padding: 0;
```

Comment:
We discussed this a bit in Slack, but just to follow up: what's the purpose of removing the padding on this button?

Reply:
Good catch. This was to adjust the count positioning (now that we're not using the translate property), but I didn't understand the intent of the padding for icon spacing. This should be fixed in 5a32a4a